README.md

```diff
@@ -324,33 +324,39 @@ Explicit:
 Unchecked
 ```
 
-Once your Google account is registered and the music is uploaded, extract the cookies for our test suite via:
+Once your Google account is registered and the music is uploaded, set the following environment variables
 
 ```
-# Enter into the node REPL
-node
-
-# Inside of the REPL, dump our cookies
-var browser = require('wd').remote();
-browser.init({browserName: 'chrome'}, console.log);
-// Wait for browser window to open
-browser.get('https://play.google.com/music/listen', console.log);
-// Wait for redirect to accounts.google.com
-// Manually log in to page
-// When you are logged in to Google Music, dump the output of the following into `test/cookies.json`
-browser.allCookies(function (err, cookies) { fs.writeFileSync('test/cookies.json', JSON.stringify(cookies, null, 4), 'utf8'); });
+GOOGLE_MUSIC_JS_USER = {your_google_account_username}
+GOOGLE_MUSIC_JS_PASSWORD = {your_google_account_password}
 ```
 
+These are required for the test suite to automate the login process to test against Google Play Music.
+
 Finally, we can run the test suite:
```

Comment:
I'm now realizing that there's no reason the 2 parts of these instructions should be separate any more. Additionally, we should not be encouraging global environment variables. Can we update the documentation to be more like:

```md
Once your Google account is registered and the music is uploaded, we are ready to run our test suite:

\```bash
# Set up Google Music credentials as environment variables
export GOOGLE_MUSIC_JS_EMAIL="your.google@account.email"
export GOOGLE_MUSIC_JS_PASSWORD="your-google-account-password"
# If you are on Windows, use the following
# set GOOGLE_MUSIC_JS_EMAIL="your.google@account.email"
# set GOOGLE_MUSIC_JS_PASSWORD="your.google@account.email"

# Start up a Selenium server
npm run webdriver-manager-start &
...
\```
```